cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-bitbucket/findings_and_ignores.json

```diff
@@ -566,5 +566,23 @@
     "supply-chain1",
     "supply-chain2",
     "supply-chain3"
+  ],
+  "contributions": [
+    {
+      "commit_hash": "sanitized",
+      "commit_timestamp": "sanitized",
+      "contributor": {
+        "commit_author_name": "Environment Test",
+        "commit_author_email": "test_environment@test.r2c.dev"
```

Comment:
Do we need to do something special when testing locally to get those git email? Or it's mocked somewhere
so we don't have to do anything?

Reply:
These are included in the commits that are created in test-setup fixtures.